modules/core/misc/java/src/java/core+Mat.java

```diff
@@ -1001,7 +1001,7 @@ public int put(int row, int col, short[] data) {
     }
 
     // javadoc:Mat::put(row,col,data)
-    public int put(int row, int col, byte[] data) {
+    public int put(int row, int col, byte[] data, int offset, int length) {
```

Comment:
Please avoid changing of existed interfaces (making already existed programs incompatible) without strict requirements.
You should add new one overload here and below for native method (or use new name).